nesta/production/batchables/unsdg/unsdg_text_predict/run.py

```diff
@@ -30,10 +35,21 @@ def dummy_model(text):
 
     return predictions
 
+def predict_sdg_labels(topic_dist, model_dictionary):
+
+    sd_labels_predictions = []
+    for sdg_label,model in model_dictionary.items():
+        prediction = model.predict(topic_dist)
+        sd_labels_predictions.extend(list(prediction))
+
+    indices = [i for i, x in enumerate(sdg_labels_predict) if x == 1]
```

Comment:
```suggestion
    sdg_list = [i+1 for i, x in enumerate(sdg_labels_predict) if x == 1]
```
You can add the extra digit directly in this list comprehension.